keploy.sh

```diff
@@ -1,184 +1,223 @@
 #!/bin/bash
 
-installKeploy (){
-    IS_CI=false
-    for arg in "$@"
-    do
-        case $arg in
-            -isCI)
-                IS_CI=true
-                shift
+# Function to display the menu and handle user selection
+platform="Linux"
+OS_NAME="$(uname -s)"
+IS_CI=false
+
+isRunningOnWindows (){
+    if [[ "$OS_NAME" == MINGW32_NT* ]] || [[ "$OS_NAME" == MINGW64_NT* ]]; then
+        echo "Windows is not supported. Please use WSL2 instead."
+        exit 1
+    elif [[ "$OS_NAME" == CYGWIN_NT* ]]; then
+        echo "Please install Linux to run this script."
+        exit 1
```

Comment:
rather than this original code was more correct as this checking is not required.